src/Symfony/Component/Security/Http/Firewall/ExceptionListener.php

```diff
@@ -131,8 +131,6 @@ private function handleAccessDeniedException(GetResponseForExceptionEvent $event
             } catch (\Exception $e) {
                 $event->setException($e);
             }
-
-            return;
```

Comment:
this now calls the accessDeniedHandler even when we call the entry point, which looks weird.

Reply:
Good point, but it's pretty much technically in line given the rest of the implementation (it also logs at 123, same thing). Not really trivial to fix given how the flow works...